Felienne/hedy
# [UI IMPROVEMENT] Improves admin views

**Description**
In this PR we make some QoL changes to the admin views. First we make sure the adventures are sorted by `last updated`, to always show the last updated adventure first. We also fix an incorrect column title and improve the styling of all admin view headers.

**Fixes**
This PR fixes #3004.

**How to test**
Verify that the adventures are now sorted by last updated, also verify that the column name is changed from `created` to `last updated`. All other changes are to simplify the styling and shouldn't influence any functionality or UI.
